pyproject.toml

```
@@ -39,10 +39,11 @@ Discord = "https://discord.gg/TPBU2sa8Et"
 [project.optional-dependencies]
 cirq = ["cirq-core>=1.3.0,<1.6.0"]
 qasm3 = ["pyqasm>=0.4.0,<1.1.0", "numpy"]
-test = ["qbraid>=0.9.0,<0.11.0", "pytest", "pytest-cov", "autoqasm>=0.1.0"]
+squin = ["kirin-toolchain>=0.17.33", "bloqade-circuit>=0.9.1"]
```

```suggestion
squin = ["kirin-toolchain>=0.22.0", "bloqade-circuit>=0.10.0"]
```
bump dependency to the latest version.